index.html

```diff
@@ -520,9 +520,116 @@
 				grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
 			}
 
+			#minimizeButton {
+				background: transparent; 
+				border: 0px;
+				width: 30px;
+				height: 30px;
+				fill: var(--item-col);
+				transition: .15s;
+			}
+
+			#minimizeButton:hover {
+				background: var(--item-col);
+				fill: var(--highlight-col);
+			}
+
+			#minMaxButton {
+				background: transparent; 
+				border: 0px;
+				width: 30px;
+				height: 30px;
+				fill: var(--item-col);
+				transition: .15s;
+			}
+
+			#minMaxButton:hover {
+				background: var(--item-col); 
+				fill: var(--highlight-col);
+			}
+
+			#closeButton {
+				background: transparent;
+				border: 0px;
+				width: 30px;
+				height: 30px;
+				fill: var(--item-col);
+				transition: .15s;
+			}
+
+			#closeButton:hover {
+				background: red;
+				fill: var(--highlight-col);
+			}
+
+			#versionNumber {
+				position: fixed; 
+				text-align: center; 
+				right: 5px; 
+				bottom: 0px; 
+				color: var(--text-main-L-col); 
+				display: inline-block;
+			}
+			#titleVersionNumber {
+				position: fixed; 
+				text-align: center; 
+				left: 5px; 
+				top: 5px; 
+				color: var(--text-main-L-col); 
+			}
+
+			#settingsInfo {
+				position: fixed;
+				font-size: 30px;
+				width: 100%;
+				text-align: center;
+				top: 50%;
+				color: var(--text-main-L-col); 
+			}
+
+			#titleBar {
+				height: 30px;
+				background: var(--menu-col);
+				-webkit-app-region: drag;
+			}
+			#titleBarContainer {
+				position: fixed;
+				top: 5px;
+				text-align: center;
+				width: 100%;
+			}
+
+			#titleBarWindowControlsContainer {
+				position: fixed;
+				right: 0%;
+				-webkit-app-region: no-drag;
+			}
+
 		</style>
 	</head>
 	<body>
+		<div id="titleBar"> <!-- use to define the height of the status bar -->
+			<div id="titleBarContainer"> <!-- use to define the location of the status bar title -->
+			<div id="titleBarName">SplitsAnalyser</div>

```

Comment:
```suggestion
				<div id="titleBarName"/>
```
Name is already handled within index.js, so unnecessary to add here again.

Reply:
thats there incase it doesnt work